EDSAnalytics/EDSTest/README.md

```diff
@@ -0,0 +1,105 @@
+# EDS Analytics Sample and Test
+
```

Comment:
Add `**Version:**: 1.0.0` between header and build status badge

Reply:
Done.